pjmedia/src/pjmedia-codec/and_vid_mediacodec.cpp

```diff
@@ -0,0 +1,1929 @@
+                              pj_size_t count,
+                              pjmedia_frame packets[],
+                              unsigned out_size,
+                              pjmedia_frame *output);
+#endif
+
+
+#if PJMEDIA_HAS_AND_MEDIA_H264
+static pj_str_t H264_sw_encoder[] = {{(char *)"OMX.google.h264.encoder\0",
+				      23}};
+static pj_str_t H264_hw_encoder[] =
+				  {{(char *)"OMX.qcom.video.encoder.avc\0", 26},
+			          {(char *)"OMX.Exynos.avc.Encoder\0", 22}};
+static pj_str_t H264_sw_decoder[] = {{(char *)"OMX.google.h264.decoder\0",
+				      23}};
+static pj_str_t H264_hw_decoder[] =
+				  {{(char *)"OMX.qcom.video.decoder.avc\0", 26},
+			          {(char *)"OMX.Exynos.avc.dec\0", 18}};
+#endif
+
+#if PJMEDIA_HAS_AND_MEDIA_VP8
+static pj_str_t VP8_sw_encoder[] = {{(char *)"OMX.google.vp8.encoder\0", 23}};
+static pj_str_t VP8_hw_encoder[] =
+				 {{(char *)"OMX.qcom.video.encoder.vp8\0", 26},
+			         {(char *)"OMX.Exynos.vp8.Encoder\0", 22}};
+static pj_str_t VP8_sw_decoder[] = {{(char *)"OMX.google.vp8.decoder\0", 23}};
+static pj_str_t VP8_hw_decoder[] =
+				 {{(char *)"OMX.qcom.video.decoder.vp8\0", 26},
+			         {(char *)"OMX.Exynos.vp8.dec\0", 18}};
+#endif
+
+#if PJMEDIA_HAS_AND_MEDIA_VP9
+static pj_str_t VP9_sw_encoder[] = {{(char *)"OMX.google.vp9.encoder\0", 23}};
+static pj_str_t VP9_hw_encoder[] =
+				 {{(char *)"OMX.qcom.video.encoder.vp9\0", 26},
+			         {(char *)"OMX.Exynos.vp9.Encoder\0", 22}};
+static pj_str_t VP9_sw_decoder[] = {{(char *)"OMX.google.vp9.decoder\0", 23}};
+static pj_str_t VP9_hw_decoder[] =
+				 {{(char *)"OMX.qcom.video.decoder.vp9\0", 26},
+			         {(char *)"OMX.Exynos.vp9.dec\0", 18}};
+#endif
+
+static struct and_media_codec {
+    int		       enabled;		  /* Is this codec enabled?	     */
+    const char	      *name;		  /* Codec name.		     */
+    const char        *description;       /* Codec description.              */
+    const char        *mime_type;         /* Mime type.                      */
+    pj_str_t          *encoder_name;      /* Encoder name.                   */
+    pj_str_t          *decoder_name;      /* Decoder name.                   */
+    pj_uint8_t	       pt;		  /* Payload type.		     */
+    pjmedia_format_id  fmt_id;		  /* Format id.   		     */
+    pj_uint8_t         keyframe_interval; /* Keyframe interval.              */
+
+    open_cb            open_codec;
+    process_encode_cb  process_encode;
+    encode_more_cb     encode_more;
+    decode_cb          decode;
+
+    pjmedia_codec_fmtp dec_fmtp;	  /* Decoder's fmtp params.	     */
+}
+and_media_codec[] = {
+#if PJMEDIA_HAS_AND_MEDIA_H264
+    {0, "H264",	"Android MediaCodec H264 codec", "video/avc",
+        NULL, NULL,
+        PJMEDIA_RTP_PT_H264, PJMEDIA_FORMAT_H264, KEYFRAME_INTERVAL,
+        &open_h264, &process_encode_h264, &encode_more_h264, &decode_h264,
+        {2, {{{(char *)"profile-level-id", 16}, {(char *)"42e01e", 6}},
```

Comment:
Is there any reason to use `42e01e`by default instead of `640028` like it done in `openh264.cpp`?

Reply:
`42e01e` also used on `openh264.cpp`  https://github.com/pjsip/pjproject/blob/master/pjmedia/src/pjmedia-codec/openh264.cpp#L284

Reply:
Sorry. Forgot that I've changed it in my version.